editoast/openapi.yaml

```diff
@@ -697,8 +698,6 @@ paths:
               schema:
                 type: array
                 items:
-                  discriminator:
-                    propertyName: type
```

Comment:
Is this deletion intended?

Reply:
Yes it's intended

I don't really know how the discriminator works for openapi definition. 
But with RTK, when there is a discriminator, it generates a field (`type` here) with the name of the object defined in the yml file (so `RouteTrackRangesNotFoundError` & `RouteTrackRangesCantComputePathError`).

But in the yml file we also define a property `type` on the object with a enum with only value (to make a constant).
So with the RTK generation we have two times the property ` type` defined, and in some cases it  breaks the typescript compilation.

So from what I understand we need to make a choice between : 
- use the discriminator, have a good name for object and don't define the type field with en anum
- or don't use discriminator, make sure to have a `type` with a constant

Linked comment :  https://github.com/DGEXSolutions/osrd/pull/3513#discussion_r1135706748